### PR TITLE
feat: eval command

### PR DESCRIPTION
**Pull request by aishenreemo**

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { ClientOptions, GatewayIntentsString } from "discord.js";
  * @description The dev server IDs. The only places where developer commands will be registered.
  * @type {string[]}
  */
-export const DEV_SERVER_IDS: string[] = ["853812920919261235", "343572980351107077"];
+export const DEV_SERVER_IDS: string[] = ["853812920919261235", "343572980351107077", "1018304559355006998"];
```
Comment (janleigh) [inline src/config.ts:7]: what server did you add

**Review by janleigh — CHANGES_REQUESTED** [review]

File: src/commands/developer/EvaluateCommand.ts [hunk 2]
```diff
@@ -1 +1,41 @@
...
+				)
+		);
+	}
+
+	public async chatInputRun(interaction: Command.ChatInputCommandInteraction) {
+		const input = interaction.options.getString("input");
+		const embed = new LatteEmbedBuilder();
+
+		try {
+			if (!DEV_SERVER_IDS.includes(interaction.user.id)) {
```
Comment (janleigh) [inline src/commands/developer/EvaluateCommand.ts:29]: bro, just add a new array on the config for developer ids 🗿

**Review by janleigh — CHANGES_REQUESTED** [review]

File: src/config.ts (hunk 1 above)
Comment (aishenreemo) [reply src/config.ts:7]: oh, i thought it was for users haha lemme change

File: src/commands/developer/EvaluateCommand.ts (hunk 2 above)
Comment (aishenreemo) [reply src/commands/developer/EvaluateCommand.ts]: ok then

**Pull request merged**